### PR TITLE
dmesg: add French translation

### PR DESCRIPTION
**Pull request by jay-tau**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/sunos/dmesg.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # dmesg
 
-> Ecrire les messages du kernel sur la sortie standard.
-> Plus d'informations : <https://www.unix.com/man-page/sunos/1m/dmesg>.
+> Écrire les messages du noyau sur la sortie standard.
+> Plus d'information: <https://www.unix.com/man-page/sunos/1m/dmesg>.
```
Comment (nicokosi) [inline pages.fr/sunos/dmesg.md:4]: I think the plural form ("informations")  is OK.
In all cases, keep the space before the semi-column:
```suggestion
> Plus d'information : <https://www.unix.com/man-page/sunos/1m/dmesg>.
```

File: pages.fr/sunos/dmesg.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
-- Afficher les messages du kernel:
+- Afficher les messages du noyau:
```
Comment (nicokosi) [inline pages.fr/sunos/dmesg.md:6]:
```suggestion
- Afficher les messages du noyau :
```

File: pages.fr/sunos/dmesg.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
 
 `dmesg`
 
-- Afficher combien de mémoire physique est disponible sur ce système:
+- Afficher la quantité de mémoire physique disponible sur ce système:
```
Comment (nicokosi) [inline pages.fr/sunos/dmesg.md:10]:
```suggestion
- Afficher la quantité de mémoire physique disponible sur ce système :
```

File: pages.fr/sunos/dmesg.md (hunk 1 above)
Comment (navarroaxel) [reply pages.fr/sunos/dmesg.md]: Our standard is `Plus d'informations :` from here https://github.com/tldr-pages/tldr/blob/main/scripts/set-more-info-link.py

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**